### PR TITLE
refactor: TECH-570 Remove all default exports from the app

### PR DESCRIPTION
**Pull request by JoakimSM**
[TECH-570](https://jira.dhis2.org/browse/TECH-570)

Even though I opened this PR, @superskip did all the work here. I think we're ready the merge this, but feel free to have a look. Great if at least one of you approve (I can't approve my own PRs 🙂 )

### CODE REVIEW

**Review by huocha — COMMENTED** [review]
Great work @superskip for such a big PR ! I have add few comments about naming, please let me know if you agree or not :)

File: src/core_modules/capture-core/rules/actionsCreator/runRulesForTEI.js [hunk 1]
```diff
@@ -95,15 +95,15 @@ function prepare(
         }
 
         const trackedEntityAttributes = getTrackedEntityAttributes(program);
-        const optionSets = optionSetsStore.get();
+        const optionSets = optionSetStore.get();
```
[the remark this answers is not in the document]
Comment (superskip) [reply src/core_modules/capture-core/rules/actionsCreator/runRulesForTEI.js:98]: I think optionSetsStore is the correct name, yes :)

File: src/core_modules/capture-core/components/DataEntry/withCleanUp.js [hunk 2]
```diff
@@ -16,6 +16,6 @@ const getCleanUpHOC = (InnerComponent: React.ComponentType<any>) =>
         );
     };
 
-export default () =>
+export const withCleanUpHOC = () =>
```
Comment (simonadomnisoru) [inline src/core_modules/capture-core/components/DataEntry/withCleanUp.js:19]: Hey, 
Do you think is necessary to add the `HOC` suffix to this component? There are other HOCs in this PR that follow a different naming pattern like `withCancelButton` or `withBrowserBackWarning`. I would suggest settling on a convention and apply it in all cases. What do you think?

**Comment by simonadomnisoru** [comment]
Hey @superskip, 
Great job on this PR 👏. 
I noticed the Netlify checks are failing. The fix for this was merged yesterday to master. Can you pull the latest changes in your PR? 
Thank you!

File: src/core_modules/capture-core/components/FormFields/UserField/Search.component.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ const exitBehaviours = {
     DO_NOTHING: 'doNothing',
 };
 
-class UserSearch extends React.Component<Props, State> {
+export class Search extends React.Component<Props, State> {
```
[the remark this answers is not in the document]
Comment (superskip) [reply src/core_modules/capture-core/components/FormFields/UserField/Search.component.js:35]: Will change to UserSearch.

File: src/core_modules/capture-core/metaData/helpers/EventProgram/getStageFromProgramId.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ const errorMessages = {
     GENERIC_ERROR: 'An error has occured. See log for details',
 };
 
-export default function getStageForEventProgram(programId: string) {
+export function getStageFromProgramId(programId: string) {
```
[the remark this answers is not in the document]
Comment (superskip) [reply src/core_modules/capture-core/metaData/helpers/EventProgram/getStageFromProgramId.js:14]: Changing back to getStageForEventProgram, and will also change the file name.

File: src/core_modules/capture-core/rules/actionsCreator/runRulesForTEI.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (superskip) [reply src/core_modules/capture-core/rules/actionsCreator/runRulesForTEI.js:98]: Discussed this on Wednesday with Joakim who thought optionSetStore was a better name after all.

File: src/core_modules/capture-core/components/DataEntry/withCleanUp.js (hunk 2 above)
Comment (superskip) [reply src/core_modules/capture-core/components/DataEntry/withCleanUp.js:19]: I think you're right, the name should be changed to withCleanUp.

**Review by simonadomnisoru — APPROVED** [review]

**Pull request merged**